### ISSUE TITLE
UCI option to set log level

### ISSUE THREAD

**Issue by sictransit** [sictransit/woodpusher#39]
It would be nice to be able to set the log level using an UCI option. As of the upcoming version the log has been pruned, but for debugging it would be good to have a more verbose log.

**Comment by tissatussa** [comment]
isn't this related to #35 ?

**Comment by sictransit** [comment]
> isn't this related to #35 ?

Yes! That's the plan, i.e. being able to set a loglevel. In the most recent version most of the chatter has been moved do the DEBUG level, but in certain circumstances it would be beneficial to enable it.
